### PR TITLE
ApiBundle fixes in composer.js

### PR DESCRIPTION
**Pull request by dawkaa**
Give exceptions, such as could not find a driver when commands create a test database. So I added the requirement in composer.js to fix it.

| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/composer.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
         "symfony/dotenv": "^5.4 || ^6.0",
         "symfony/http-client": "^5.4 || ^6.0",
         "symfony/webpack-encore-bundle": "^1.15",
-        "theofidry/alice-data-fixtures": "^1.4"
+        "theofidry/alice-data-fixtures": "^1.4",
+        "ext-sqlite3": "*"
```
Comment (jakubtobiasz) [inline src/Sylius/Bundle/ApiBundle/composer.json]: Keep alphabetical order please :).

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]
Also rebase on top of current 1.13 branch as we fixed the root of problem with failing tests.

File: src/Sylius/Bundle/ApiBundle/composer.json (hunk 1 above)
Comment (dawkaa) [reply src/Sylius/Bundle/ApiBundle/composer.json]: yes i see @jakubtobiasz, and fixed it

**Comment by dawkaa** [comment]
i rebased and pushed

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thanks, @dawkaa! :tada:

**Comment by dawkaa** [comment]
> 

😁😁

**Pull request merged**